can/exceptions.py

```diff
@@ -0,0 +1,37 @@
+"""
+Exception classes.
+"""
+
+class CanError(Exception):
+	""" Base class for all can related exceptions.
+	"""
+	pass
+
+
+class CanBackEndError(CanError):
+	""" Indicates an error related to the backend (e.g. driver/OS/library)
+	Examples:
+	- A call to a library function results in an unexpected return value

```

Comment:
Should we add the return code as an parameter, which is `None` if not explicitly set? Also applies to CanInitializationError. This way, we could unify it a little bit, and it would be opt-in for interfaces where it would be helpful to have the error codes as a user.